components/accordion-menu/script.js

```diff
@@ -137,6 +137,7 @@ class AccordionHeader extends HTMLElement {
         this._caret.classList.add('guic-accordion-header-caret');
 
         // Clear any carets on rerender
+        // eslint-disable-next-line no-self-assign
         this.textContent = this.textContent;
```

Comment:
Why do we need to self-assign the variable here? :D Is there a problem if we don't do it?

Reply:
Two carets appear on the accordion so we must do it.